prime-dotnet-webapi/Configuration/Database/Seeded Tables/LicenseDetailConfiguration.cs

```diff
@@ -50,6 +54,9 @@ public override IEnumerable<LicenseDetail> SeedData
                     new LicenseDetail { Id = 29, LicenseCode = 29, Prefix = "T9", Manual = false, Validate = false, NamedInImReg = false, LicensedToProvideCare = true,  EffectiveDate = SEEDING_DATE.DateTime.ToUniversalTime(), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
                     new LicenseDetail { Id = 31, LicenseCode = 31, Prefix = "T9", Manual = true,  Validate = false, NamedInImReg = false, LicensedToProvideCare = false, EffectiveDate = SEEDING_DATE.DateTime.ToUniversalTime(), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
                     new LicenseDetail { Id = 68, LicenseCode = 68, Prefix = "T9", Manual = true,  Validate = false, NamedInImReg = false, LicensedToProvideCare = true,  EffectiveDate = SEEDING_DATE.DateTime.ToUniversalTime(), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new LicenseDetail { Id = 70, LicenseCode = 29, Prefix = "T9", Manual = false, Validate = true,  NamedInImReg = true,  LicensedToProvideCare = true,  EffectiveDate = IMREG_UPDATE1_DATE.DateTime.ToUniversalTime(), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
```

Comment:
The existing records call `SEEDING_DATE.DateTime.ToUniversalTime()` since `SEEDING_DATE` is shared across our tables. In the case of this new effective date, there is no reason to have it be a `DateTime` and then convert it to UTC everywhere. Either parse it as a UTC `DateTime` or do the conversion once on line 10.